frontend/src/components/activity/ScheduleEntry.vue

```diff
@@ -192,6 +192,16 @@ Displays a single scheduleEntry
                 </tr>
               </tbody>
             </table>
+            <DialogActivityEdit
```

Comment:
After the dialog is closed, you have to refresh the scheduleentries of the activity.
(and the title and the category of the activity).
But refreshing everything would be a little too much